docs/config.rst

```diff
@@ -110,3 +110,42 @@ Nox will call your session functions with a :class:`Session` object. You use thi
 .. autoclass:: Session
     :members:
     :undoc-members:
+
+
+Modifying Nox's behavior in the Noxfile
+---------------------------------------
+
+Nox has various :doc:`command line arguments <usage>` that can be used to modify its behavior. Some of these can also be specified in the Noxfile using ``nox.options``. For example, if you wanted to store Nox's virtualenvs in a different directory without needing to pass it into ``nox`` every time:
```

Comment:
You're a fan of infinite line length?

Reply:
I docs I don't give a crap. It's obnoxious to constantly reflow prose.